### ISSUE TITLE
Cannot Train on Pascal VOC

### ISSUE THREAD

**Issue by ManishSoni1908** [fizyr/keras-retinanet#979]
Training loss is decreasing but validation map comes out to zero for all classes.
Please help me out.
Thanks

**Comment by hodaatef** [comment]
me too the same problem

**Comment by widedh** [comment]
Did you find a solution?

**Comment by wangchau** [comment]
I have the same question, did you solve this problem?
Thanks 
@ManishSoni1908 @hodaatef @widedh

**Comment by hodaatef** [comment]
Train using coco weights as initial weights

**Comment by hgaiser** [comment]
> Train using coco weights as initial weights

This is a good tip, but also, if your mAP stays 0 it usually indicates your dataset is incorrect. Could be a misunderstanding of the format of the CSV dataset (for example I've seen cases where the bounding boxes were encoded as "x, y, w, h" while they should be "x1, y1, x2, y2"). Use the [debugging tool](https://github.com/fizyr/keras-retinanet#debugging) to visualize your dataset.

**Comment by waylonflinn** [comment]
Debugging tool is undocumented (as far as I can tell). 
Here's a command line that will visualize your annotations in PASCAL VOC format.

Change `trainval` to be the name of the text file (in `ImageSets/Main`) with your image name list.
Images must be in a folder named `JPEGImages` and end with `.jpg`.
```
python3 debug.py --annotations pascal --pascal-set trainval /path/to/pascal
```

COCO
Directory layout is different for COCO. I'm only providing a possible command line here.
```
python3 debug.py --annotations coco --coco-set val2017 /path/to/coco
```


Script can be found at `keras_retinanet/bin/debug.py`
